pyrevolve/pyrevolve.py

```diff
@@ -163,3 +153,59 @@ def save_checkpoint(self):
     def load_checkpoint(self):
         locations = self.checkpoint.get_data_location(self.scheduler.capo)
         self.storage.load(self.scheduler.cp_pointer, locations)
+
+
+class DiskRevolver(BaseRevolver):
+    """
+        This class is an specialization of the Revolver class
+        that uses a DiskStorage as its default storage method.
+        When no 'filename' is provided, the storage
+        .dat file is created inside the working directory.
+        If the 'remove_storagefile' is set to True (default),
+        the storage file is removed by default when storage object
+        is destroyed.
+    """
+    def __init__(self, checkpoint, fwd_operator, rev_operator,
+                 n_checkpoints=None, n_timesteps=None, timings=None,
+                 filename=None, remove_storagefile=True):
```

Comment:
Wouldn't it be better to take a directory instead of a filename? Then we can try to generate unique filenames instead of leaving that to the user?
Also, is the remove_storagefile flag necessary? Why not just remove it every time?

Reply:
Agreed.